pkg/api/common/versions.go

```diff
@@ -120,9 +120,10 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.14.0-beta.2":  false,
 	"1.14.0-rc.1":    false,
 	"1.14.0":         false,
-	"1.14.1":         true,
+	"1.14.1":         false,
```

Comment:
same comment here about not disabling until Azure Stack supports 1.14.4

Reply:
Yep. 👍